cascade-api/test/Test/Cascade/Api/StateMachine/Command/Task.hs

```diff
@@ -0,0 +1,610 @@
+      execute input = evalIO . Cascade.Api.Tasks.deleteById $ id
+          where id = input ^. #id . concreted
+
+      ensure
+        :: Model Concrete
+        -> Model Concrete
+        -> DeleteById Concrete
+        -> Cascade.Api.Tasks.DeleteByIdResponse
+        -> Test ()
+      ensure _before _after input response = do
+        footnoteShow response
+        task :: Task.Readable <-
+          (response ^. #responseBody)
+          |> matchUnion @(Response.Ok Task.Readable)
+          |> coerce
+          |> evalMaybe
+        task ^. #id === input ^. #id . concreted
+
+      update
+        :: Ord1 v
+        => Model v
+        -> DeleteById v
+        -> Var Cascade.Api.Tasks.DeleteByIdResponse v
+        -> Model v
+      update model (DeleteById id) _response =
+          model |> #task . #creatables . traversed %~ sans id
+
+      require :: Model Symbolic -> DeleteById Symbolic -> Bool
+      require model (DeleteById id) =
+          case
+              model
+              ^.. #task
+              .   #creatables
+              .   folded
+              .   at id
+              .   _Just
+              |>  listToMaybe
+            of
+              Nothing -> False
+              Just _  -> True
+  in  Command generator execute [Require require, Update update, Ensure ensure]
+
+deleteNotExisting
+  :: forall g m . MonadGen g => MonadIO m => MonadTest m => Command g m Model
+deleteNotExisting =
+  let generator :: Model Symbolic -> Maybe (g (DeleteById Symbolic))
+      generator model = case model ^. #task . #notExistingIds of
+        []  -> Nothing
+        ids -> Gen.element ids |> fmap DeleteById |> Just
+
+      execute :: DeleteById Concrete -> m Cascade.Api.Tasks.DeleteByIdResponse
+      execute input = evalIO . Cascade.Api.Tasks.deleteById $ id
+          where id = input ^. #id . concreted
+
+      ensure
+        :: Model Concrete
+        -> Model Concrete
+        -> DeleteById Concrete
+        -> Cascade.Api.Tasks.DeleteByIdResponse
+        -> Test ()
+      ensure _before _after _input response = do
+        footnoteShow response
+        response ^. #responseStatusCode . #statusCode === 404
+  in  Command generator execute [Ensure ensure]
+
+updateCreatableTask
+  :: Task.RawUpdatable -> Task.RawCreatable -> Task.RawCreatable
+updateCreatableTask updatable Task.RawCreatable {..} = Task.RawCreatable
+  { title      = fromMaybe title $ updatable ^. #title
+  , deadlineAt = fromMaybe deadlineAt $ updatable ^. #deadlineAt
+  }
+
+checkEqReadableRawCreatableTask
+  :: (MonadTest m, HasCallStack) => Task.Readable -> Task.RawCreatable -> m ()
+checkEqReadableRawCreatableTask task creatable = do
+  task ^. #title . to Text.NonEmpty.un === creatable ^. #title
+  task
+    ^.  #deadlineAt
+    .   to unFormattedOffsetDatetime
+    .   to offsetDatetimeToTime
+    === creatable
+    ^.  #deadlineAt
+    .   to unFormattedOffsetDatetime
+    .   to offsetDatetimeToTime
```

Comment:
Ok! This the worst formatting in the universe. Thank you, Brittany!
Does adding some parens solve the issue?